### DISCUSSION TITLE
Добавить поддержку Roistat в формы Elementor

### DISCUSSION THREAD

**Discussion by kavabangaseo** [mihdan/events-tracker-for-elementor#14] [Ideas]
Привет
Прошу добавить поддержку Roistat на формах elementor
Вот документация Roistat:
Для Contact Form 7 https://help.roistat.com/integrations/CRM_i_CMS/Prochee/WordPress_%28Contact_Form_7%29/
И то что обычно используем для сайтов это просто через скрипт Roistat мы размечаем форму Elementor как на скриншоте:
<img width="680" alt="Снимок экрана 2020-12-14 в 21 55 48" src="https://user-images.githubusercontent.com/76007906/102122941-4bd10b00-3e57-11eb-8838-46e6a8729fc5.png">
В идеале конечно чтобы в настройках формы можно было бы выбрать нужные параметры и названия input

**Comment by kavabangaseo** [comment]
Вот инструкция для настройки передачи из форм данных в Roistat: https://help.roistat.com/integrations/CRM_i_CMS/Dopolnitel’nye_nastroiki_integracii/Peredacha_zajavok_v_CRM_cherez_Roistat_%28proksilid%29/#_1

**Comment by mihdan** [comment]
@kavabangaseo  Выглядит интересно, сколько вы готовы заплатить за реализацию данного функционала?

**Reply by kavabangaseo** [reply]
Не оценивал если честно, пару раз у клиентов вопросы возникали по интеграции и стали чаще
Так что пока просто предложил
Попробую понять сколько стоит интеграция и тогда пойму по вашему вопросу)
